tilepack/builder.py

```diff
@@ -100,14 +101,19 @@ def build_tile_packages(min_lon, min_lat, max_lon, max_lat, min_zoom, max_zoom,
             t.add_metadata('name', output)
             # FIXME: Need to include the `json` key
             t.add_metadata('format', 'application/vnd.mapbox-vector-tile')
+            if typ == 'vector':
+                t.add_metadata('compression', 'deflate')
             t.add_metadata('bounds', ','.join(map(str, [min_lon, min_lat, max_lon, max_lat])))
             t.add_metadata('minzoom', min_zoom)
             t.add_metadata('maxzoom', max_zoom)
 
         for format_args, response_info, data in p.imap_unordered(fetch_tile, fetches):
             for t in tile_ouputters:
                 if data:
-                    t.add_tile(format_args, data)
+                    if typ == 'vector':
```

Comment:
Rather than testing for `typ == 'vector'` to decide if we should use compression, please pass in a `compression` arg so that we can decide whether or not to compress independent of which tile type to download.